tests/test_v2.py

```diff
@@ -22,3 +22,16 @@ def test_v2_functional_time_value_zero(tmp_path):
 
     assert local_run.main(tmp_path, ["run_all"])
     assert (tmp_path / "output/id/report.html").exists()
+
+
+def test_v2_functional_time_ever(tmp_path):
+    analysis = v2.Analysis(**v2.TEST_DEFAULTS)
```

Comment:
In the `TEST_DEFAULTS`, `time_ever` is being set to False. However, I think Tom said it would either be True or None. It works as-is, but it could be misleading in future.